az-hadoop-jobtype-plugin/src/main/java/azkaban/jobtype/AbstractHadoopJavaProcessJob.java

```diff
@@ -62,10 +62,6 @@ public AbstractHadoopJavaProcessJob(String jobid, Props sysProps, Props jobProps
   public void run() throws Exception {
     try {
       super.run();
-    } catch (Throwable t) {
-      t.printStackTrace();
-      getLog().error("caught error running the job", t);
-      throw t;
```

Comment:
Are these 2 causing the NPE?

Reply:
No, the NPE is thrown when creating Noop jobs because private properties is null. Fix is in JobTypeManager lines 466-471.